core/src/avm2/globals/avmplus.as

```diff
@@ -109,6 +109,9 @@ package avmplus {
     }
 
     public function describeType(value: *, flags: uint):XML {
+        if (value == null) {
+            return <type name="null" isDynamic="false" isFinal="true" isStatic="false"/>;
```

Comment:
From what I can see this should actually be implemented in `describeTypeJSON`. Try comparing the output of `describeType(null,INCLUDE_BASES)` between this branch and FP (with `avmplus.describeType` and `avmplus.INCLUDE_BASES`)